osbenchmark/utils/parse.py

```diff
@@ -22,7 +22,7 @@ def parse_string_parameter(key: str, params: dict, default: str = None) -> str:
 
 def parse_int_parameter(key: str, params: dict, default: int = None) -> int:
     if key not in params:
-        if default:
+        if default is not None:
```

Comment:
This change is technically not related to this PR, but the previous `if` statement was flaky if `default == 0`. All unit tests pass in both configurations.